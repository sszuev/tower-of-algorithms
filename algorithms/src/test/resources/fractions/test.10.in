1/22 + 4/31 + 6/10+3/25