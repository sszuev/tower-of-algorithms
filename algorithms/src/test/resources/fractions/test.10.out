15251/17050